cmd/mattermost/commands/config_test.go

```diff
@@ -71,7 +74,11 @@ func TestConfigValidate(t *testing.T) {
 	th := Setup()
 	defer th.TearDown()
 
-	assert.Error(t, th.RunCommand(t, "--config", "foo.json", "config", "validate"))
```

Comment:
Instead of using a path to a file that doesn't exist -- which is actually a valid way to "start" the server, create one and validate the invalidly empty file as a more meaningful test.